WORKSPACE

```diff
@@ -66,7 +66,7 @@ maven_install(
     name = "maven",
     artifacts = [
         "antlr:antlr:2.7.7",
-        "org.apache.zookeeper:zookeeper:3.4.14",
+        "org.apache.zookeeper:zookeeper:3.5.8",
```

Comment:
The `maven_install.json` pin file needs to be updated.

Reply:
We should probably add these instructions to the documentation somewhere.

Reply:
Yup. And maybe above the list of dependencies instead of below them.